src/System.Net.NetworkInformation/src/System/Net/NetworkInformation/OsxIPGlobalProperties.cs

```diff
@@ -122,7 +146,7 @@ public override IPGlobalStatistics GetIPv6GlobalStatistics()
         public override TcpStatistics GetTcpIPv4Statistics()
         {
             // OSX does not provide separated TCP-IPv4 and TCP-IPv6 stats.
-            throw new PlatformNotSupportedException(SR.net_InformationUnavailableOnPlatform);
+            return new OsxTcpStatistics();
```

Comment:
Maybe I missed it... why was this throwing previously if we already had the implementation for it?

Reply:
OSX (and FreeBSD) does not track transport (TCP/UDP) statistic per network protocol (IPv4/6)
For UDP we return same values for both IPv4 and IPv6. For TCP we did return common stats for IPv6 but not for IPv4. That popped when I added basic tests and it looked wrong. 

This change makes it consistent even if not 100% correct. Alternatively we can throw for all four cases but that does not seems productive.